### DISCUSSION TITLE
LAB 03.01 Task02

### DISCUSSION THREAD

**Discussion by JuanDavidSanchezA** [rramosp/ai4eng.v1#170] [03 - GENERACION DE MODELOS]
Hola, he desarrollado este código

![image](https://user-images.githubusercontent.com/108543458/179845674-e73755e5-4e71-4d3f-aaa8-811dc9f58e20.png)

Pero al ejecutarlo me arroja este error

![image](https://user-images.githubusercontent.com/108543458/179846288-23689edf-5099-48bd-95e0-a3c9c84d73ae.png)

No entiendo por qué si la librería pandas ya se importó al inicio

![image](https://user-images.githubusercontent.com/108543458/179846749-02bb74de-67c3-4224-b6a0-515a17d285c2.png)

Incluso lo intenté con import pandas as pd en la misma celda donde se define la función y salió el mismo error

**Comment by JuanDavidSanchezA** [comment]
Ya encontré la forma, debía escribir
def fit(X,y):
  import pandas as pd

así si funciona, debía importar la librería dentro de la definición de la función

**Comment by Felipe-RA** [comment]
Saludos JuanDavidSanchezA

Me alegra que lo hayas descubierto, gracias por compartir!

Hasta pronto!
